### PR TITLE
Refactor model

### LINKED ISSUE #52: Data model refactoring
The data model of the application needs to be refactored:
- translate data model to english
- make the data model more general, as currently the model is mostly designed for Bucharest

### PR DESCRIPTION
**Pull request by tudoramariei**
Fixes #52 

Changes:
- translate `map_app/model.py` from Romanian to English in:
  - model
  - migrations
  - test JSON
  - test XLSX
- remove 'nr_sector' from xlsx
- remove `*.xlsx` & `*.csv` from `gitignore` and add the temporary `~*.xlsx`
- apply black formatting to all files worked on

### CODE REVIEW

**Comment by iuliux** [comment]
A couple of things I noticed while scanning the changes:

1. Isn't refactoring migration files a really bad idea?
2. Don't really want to get into that conversation, but just curious why changing to double-quotes?

**Comment by tudoramariei** [comment]
> 1. Isn't refactoring migration files a really bad idea?

I'm pretty sure it is, I'll probably redo that part to make sure things are OK
Right now I'm trying to get feedback on the bigger decisions of this PR like...

- the names used for variables
- the entire removal of `nr_sector`

> 2. Don't really want to get into that conversation, but just curious why changing to double-quotes?

It's not my decision entirely; I've applied [black](https://black.readthedocs.io/en/stable/) formatting to all files I worked on
`black --line-length=80 --target-version=py37`

In that idea, I think it would be good to have a formatting tool like black just _automagically_ work on our code to make it more uniform and cleaner
We can work out the details later, but you can look at this also as a POC on how black formatting would look like

**Comment by tudoramariei** [comment]
After more Git tricks than there should have been, the PR is done and ready for checking

File: seismic_site/map_app/models.py [hunk 1]
```diff
@@ -26,46 +22,52 @@ class SeismicCategoryChoice(Enum):
...
-    nr_apart = models.IntegerField(null=True)
-    arie_desfasurata = models.FloatField(null=True)
-    an_expertiza = models.IntegerField(null=True)
-    expert_atestat = models.CharField(max_length=100)
-    obs = models.CharField(max_length=1000)
-    numar_cadastral = models.IntegerField(null=True)
-    nr_carte_funciara = models.CharField(max_length=50)
-    actualizare_pmb = models.DateField(null=True, blank=True)
-    editare_admin = models.DateField(null=True, blank=True)
+    lon = models.FloatField(null=True)
```
Comment (iuliux) [inline seismic_site/map_app/models.py]: `lng` is commonly used

Comment (tudoramariei) [reply seismic_site/map_app/models.py]: never heard of this one until now, but if other SW makers use it, then OK; I'll try to make the changes tonight

Comment (tudoramariei) [reply seismic_site/map_app/models.py]: It's generally not good to edit the migrations but I've made an exception this time :)

**Comment by tudoramariei** [comment]
@catileptic out of all the fields from the model, `nr_sector` was the only one I couldn't change to anything else; anyway, the data from it is stored into `locality` (former `sector`)
If we want to make the app national, we can't work with that field

File: seismic_site/map_app/models.py (hunk 1 above)
Comment (iuliux) [reply seismic_site/map_app/models.py]: It's a newly added migration so it's only bad for your env  😄

**Pull request merged**